sources/world/OpenStreetMap-FrenchStyle.geojson

```diff
@@ -0,0 +1,16 @@
+{
+    "type": "Feature",
+    "properties": {
+        "url": "http://{switch:a,b,c}.tile.openstreetmap.fr/osmfr/{zoom}/{x}/{y}.png",
+        "attribution": {
+            "url": "http://openstreetmap.org/",
+            "text": "© OpenStreetMap contributors, CC-BY-SA",
+            "required": false
```

Comment:
because I did like the GermanStyle....but you're right, it should be required, I just changed it